### PR TITLE
Mall tweaks

### PR DESCRIPTION
**Pull request by nextlevelbanana**

### CODE REVIEW

File: server/src/rooms/data/roomData.json [hunk 1]
```diff
@@ -219,7 +227,15 @@
...
-    "roomId": "arcade"
+    "roomId": "arcade",
+    "hasNoteWall": true,
+    "noteWallData": {
+      "roomWallDescription": "Adjacent to a screen showing recent high scores, a curious screen displays what may not be a game at all, but is rather labelled \"UNCONFERENCE TOPIC SUGGESTIONS!\" A small sign clarifies \"Write what you want to chat with others about, and upvote topics you find interesting. Moderators will assign the top six topics rooms, 5 minutes into each unconference block. Have fun!\"",
```
Comment (nextlevelbanana) [inline server/src/rooms/data/roomData.json:233]: ...will assign the top *four* topics rooms...

File: server/src/rooms/data/roomData.json [hunk 2]
```diff
@@ -547,9 +575,9 @@
...
+    "displayName": "Theatre",
+    "shortName": "the theatre",
+    "description": "Passing underneath a swirling circular Cinemaplex logo, you are confronted by an impossibly-large screen dominating one entire wall of the space. Before it is a small field of mismatched chairs visibly pulled from anywhere a spare could be found.<br/><br/>You can return to the [[information obelisk->obelisk]]. Or if you'd like to speak to one of our speakers after their talk, you can head to breakout rooms: [[Screen 1->screen1]], [[Screen 2->screen2]], [[Screen 3->screen3]], or [[Screen 4->screen4]]. (Check the \"Happening Now\" button on the left for speaker room assignments!)<br/><br/><a href=\"stream.html\" onClick=\"window.open('stream.html#' + window.getComputedStyle(document.body).getPropertyValue('background-color'), 'stream', 'width=560,height=460'); return false\">Pop Out Stream</a>. <a href=\"https://www.streamtext.net/player?event=RoguelikeCelebration \" target=\"_blank\">Pop Out Live Captions</a>.<br/>",
```
Comment (nextlevelbanana) [inline server/src/rooms/data/roomData.json:580]: clobbered change: the movie names in the nav links

Comment (apepers) [reply server/src/rooms/data/roomData.json:580]: ahhh glad you caught this!!!

**Review by apepers — APPROVED** [review]
Qristy reviewed this so approving on her behalf!

**Pull request merged**